ocsp/responder.go

```diff
@@ -49,6 +51,59 @@ func (src InMemorySource) Response(request *ocsp.Request) (response []byte, pres
 	return
 }
 
+// DBSource represnts a source of OCSP responses backed by certdb
+type DBSource struct {
+	Accessor certdb.Accessor
+}
+
+// NewDBSource creates a new DBSource type with associated dbAccessor
+func NewDBSource(dbAccessor certdb.Accessor) Source {
+	return DBSource{
+		Accessor: dbAccessor,
+	}
+}
+
+// Response implements cfssl.ocsp.responder.Source, returning the OCSP response
```

Comment:
The comment should also explain the meaning of the return arguments: what does the bool indicate? What's in the byte slice?